LICENSE.md

```diff
@@ -0,0 +1,7 @@
+Copyright (c) 2013-2015
```

Comment:
I wasn't sure who to place as copyright owner.  Nicolas is the first committer, Kohsuke has committed many times, and I've committed many times as well.

Any pointers to rules and regulations which define the copyright owner?  Is it the original author?